cmd/config.go

```diff
@@ -54,17 +54,20 @@ func configFlagSet() *pflag.FlagSet {
 	flags.Bool("no-usage-report", false, "don't send anonymous stats to the developers")
 	flags.Bool("no-thresholds", false, "don't run thresholds")
 	flags.Bool("no-summary", false, "don't show the summary at the end of the test")
+	reportSummaryUsage := "output summary to file in json format, empty string means output to standard out"
+	flags.StringP("report-summary", "r", "", reportSummaryUsage)
```

Comment:
it was previously `report` and ... I am not certain that `-summary` adds any information ... so maybe just leave it as it is. 

Also not certain if anyone would want this on the stdout as we are currently spewing way too much stuff on it and it's probably hard to parse it from there

Reply:
I'm fine with leaving the option to have stdout as an output, since the amount of other output there when `-q` is a separate issue. I have other issues with the CLI flag though:
- since this is a "_[JSON]_ report of the _[end of test metric]_ summary", not a "[???] summary of the [???] report", I think the option should be called `--summary-report`, not the current `--report-summary`
- this probably won't be a feature that's used very often, so it definitely doesn't deserve a CLI short flag (for now), especially not a single-letter one, so please remove the option to specify this as `-r` for now

Reply:
Yeah but ... let's add the ability to output to stdout **when** we fix the other issues as if someone tries to use it will just ... break. 
 I think it will be VERY frustrating to run an hour long test, pipe it's output to somewhere and for it to tell you that the input is not JSON ... even after you set `-q`

Reply:
hmm OK, :+1 for having this feature when it would be useful, since we can then do it in a backwards compatible way (i.e. currently empty string should be an error) in the future, and since users who really want to output to sdtout could still do it if we require a filename by just passing `/dev/stdout`